doc/configuration.rst

```diff
@@ -481,18 +481,21 @@ We recommend that you copy this set of exit codes to your ``luigi.cfg`` file:
   missing_data=20
   task_failed=30
   scheduling_error=35
+  unknown_reason=38
   unhandled_exception=40
 
 unhandled_exception
   For internal Luigi errors.  Defaults to 4, since this type of error
   probably will not recover over time.
+unknown_reason
+  For when a task fails or is left pending for unknown reasons.
```

Comment:
By "forgotten" I meant "pruned", yeah. If a worker loses connection intermittently for a certain time (and there are no redundant workers triggering the same workflow), then it ends up with parts of its task graph pruned and never given to it in `get_work()`, translated to "pending because of unknown reason" in summary.